lib/presentation/crowdaction/crowdaction_details/crowdaction_details_screen.dart

```diff
@@ -168,45 +173,6 @@ class _CrowdActionDetailsPageState extends State<CrowdActionDetailsPage> {
                         height: 20,
                       ),
                     ),
-                    // GestureDetector(
```

Comment:
Should this really be removed, since it will be implemented after the MVP?
Maybe annotate with "TODO implement after MVP" instead.

Reply:
There is already a TODO at the `RealtimeParticipationText`

Reply:
Yes, but I am not sure if all of this code should be removed. We will probably want to use most of it later.